scripts/run_frontend_tests.py

```diff
@@ -176,7 +176,7 @@ def main(args: Optional[Sequence[str]] = None) -> None:
                 parsed_args.download_combined_frontend_spec_file):
                 download_task = subprocess.Popen(
                     ['wget',
-                    'http://localhost:9876/base/core/templates/' +
+                    'http://localhost:9876/base/core/templates/'
                     'combined-tests.spec.js',
```

Comment:
ditto: indent with parentheses to make it clear that these two lines are a single list entry